### PR TITLE
Add docs links for common components. Fixes #212

### LINKED ISSUE #212: Add help for common/default components
Re https://github.com/aframevr/aframe-editor/issues/31 I created a PR (https://github.com/aframevr/aframe-editor/pull/211) to add the help link for each core components. In the future we'll need to add that feature to the third party components too.
But right now the default components are shown inside the `common` panel and they don't have link to the doc.

### PR DESCRIPTION
**Pull request by fernandojsg**
Added help icon button next to the common components (position, rotation, scale, visible)
screenshot 2016-08-02 23 45 56

### CODE REVIEW

File: src/actions/component.js [hunk 1]
```diff
@@ -32,6 +34,35 @@ export function getClipboardRepresentation (entity, componentName) {
...
+ * @param  {string} componentName Component's name
+ * @return {string}               URL to the documentation
+ */
+export function getComponentDocsUrl (componentName) {
+  if (AFRAME.components[componentName]) {
+    // Returns link from the core components
+    return 'https://aframe.io/docs/' + AFRAME.version + '/components/' +
+      (componentName === 'camera' ? '' : componentName.toLowerCase() + '.html');
+  }
+  return null;
```
Comment (ngokevin) [inline src/actions/component.js:48]: Need to return null?

File: src/components/components/CommonComponents.js [hunk 2]
```diff
@@ -42,8 +42,8 @@ const CommonComponents = props => {
             const schema = AFRAME.components[key].schema;
             return (
               <PropertyRow onChange={updateEntity} key={key} name={key}
-                schema={schema} data={componentData.data} componentname={key}
-                entity={props.entity}/>
+                show_help={true} schema={schema} data={componentData.data}
```
Comment (ngokevin) [inline src/components/components/CommonComponents.js:45]: camelCase?

**Comment by ngokevin** [comment]
The help links could be less pronounced (maybe some opacity).

**Comment by ngokevin** [comment]
r+wc

File: src/components/components/CommonComponents.js (hunk 2 above)
Comment (fernandojsg) [inline src/components/components/CommonComponents.js:45]: yep right.

**Comment by fernandojsg** [comment]
@ngokevin what do you think about something like this (`opacity: 0.4` and `1` on hover)
screenshot 2016-08-03 00 29 59

**Comment by ngokevin** [comment]
I like it.

File: src/actions/component.js (hunk 1 above)
Comment (fernandojsg) [inline src/actions/component.js:48]: It could be `null, false, undefined...` something to easy check against that it has a valid url.

Comment (ngokevin) [inline src/actions/component.js:48]: I mean if you get rid of this line, it will return undefined by default, which seems suitable.

Comment (fernandojsg) [inline src/actions/component.js:48]: yep, you're right, done.

**Comment by fernandojsg** [comment]
@ngokevin perfect! thank you :)

**Pull request merged**